sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/util/CosmosPagedFlux.java

```diff
@@ -203,7 +212,13 @@ private Flux<FeedResponse<T>>  wrapWithTracingIfEnabled(CosmosPagedFluxOptions p
 
                             break;
                         case ON_NEXT:
-                            this.recordFeedResponse(pagedFluxOptions, tracerProvider, response, feedResponseConsumerLatencyInNanos);
```

Comment:
what about the ERROR case?

Reply:
line 237 (after changes)

Reply:
I meant, don't we need to call the recordFeedResponse in case of ON_ERROR signal?

Reply:
No - because we do not have a FeedRespoinse but an Exception - and that is passed into endSpan

Reply:
Makes sense, thanks!